packages/cli/test/integration/credentials.ee.test.ts

```diff
@@ -15,10 +15,7 @@ import * as utils from './shared/utils';
 jest.mock('../../src/telemetry');
 
 // mock whether credentialsSharing is enabled or not
-const mockIsCredentialsSharingEnabled = jest.spyOn(
-	CredentialHelpers,
-	'isCredentialsSharingEnabled',
-);
+const mockIsCredentialsSharingEnabled = jest.spyOn(UserManagementHelpers, 'isSharingEnabled');
```

Comment:
Now that we have generalized:
`// mock whether sharing is enabled or not`
`const mockIsSharingEnabled`

Same for other spots.

Reply:
Do we need the pointer? Else `jest.spyOn(UserManagementHelpers, 'isSharingEnabled').mockReturnValue(false)`